spec/factories/important_notice.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :important_notice do
+    patient
+    team_id { create(:team).id }
+
+    notice_type { :deceased }
```

Comment:
I would suggest not having a default notice type and require uses to specify one to avoid always testing against the same type by accident. Alternatively, if we do want a default we could make it a random sample of all the types to improve the chance of catching potential issues.